src/utils/graphql/queries/getTriggers.query.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getTriggers($serial: String!) {
+    applicationTriggerStates(condition: { serial: $serial }) {
```

Comment:
Maybe use the ApplicationBySerial here and just return the required fields.

Reply:
Not sure what you mean -- this is querying a seperate View table that collates ALL trigger fields.